build.gradle

```diff
@@ -9,11 +9,12 @@ repositories {
 
 dependencies {
 	compile "com.google.code.gson:gson:2.3"
-
+	compile "org.apache.commons:commons-lang3:3.0"
```

Comment:
Good point - do you want me to add a similar method like `isNotBlank(path)` to the project's `Utils` class instead?